content/_data/plugins.json

```diff
@@ -603,6 +603,12 @@
           "link": "https://github.com/ling1726/cypress-forced-colors",
           "keywords": ["forced colors", "high contrast", "high contrast mode", "contrast", "testing"],
           "badge": "community"
+        },
+        {
+          "name": "cy-verify-downloads",
+          "description": "Cypress custom command to wait and verify that file was downloaded",
+          "link": "https://github.com/elaichenkov/cy-verify-downloads",
+          "keywords": ["commands", "assertions", "wait", "verify", "download"],
```

Comment:
Hi @elaichenkov,

This json is invalid with a trailing comma here. Also, could you add the community badge ("badge": "community")?

Thanks

Reply:
Hey,
Oh, no, sorry. Lost the badge property, after resolving merge conflicts here. I've fixed it.